src/app/services/theme-switcher.service.ts

```diff
@@ -54,7 +52,7 @@ export class ThemeSwitcherService {
   }
 
   get getActiveTheme(): Theme {
-    const activeTheme: string = localStorage.getItem(this.localStorageKey) || this.preferredBrowserTheme;
```

Comment:
Why did you remove the local storage access? This line should be

```ts
const activeTheme: string = localStorage.getItem(this.localStorageKey) || Theme[this.getPreferredBrowserTheme];
```

User-configured app settings should be persistent, after all.

Reply:
This change actually resulted in an error, which is why I fixed it myself.